frontend/src/lib/stores/cookies.ts

```diff
@@ -1,12 +1,38 @@
 import { browser } from "$app/environment"
 import { writable } from "svelte/store"
 
+export interface CookieSelection {
+  allowPreference: boolean
+  allowStatistical: boolean
+}
+
+export interface CookieType {
+  name: string
+  type?: string
+  description: string
+  cookieAmount: number
```

Comment:
What is this used for?

Reply:
It's used for typing Necessary, Statistical, etc. cookies in the file above

Reply:
I'm talking about the numbers, what do we use the amount for?

Reply:
Removed for now